plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -1,159 +1,159 @@
+      </runtime-component-type>
+      <runtime-component-version
+            type="com.google.appengine.standard.runtime"
+            version="1">
+      </runtime-component-version>
       <supported>
-          <runtime-component id="com.google.cloud.tools.eclipse.runtime.component.custom" version="1"/>
-          <facet id="jst.web" version="[2.2-2.5]"/>
-          <facet id="jst.utility"  version="1.0"/>
-          <facet id="jst.webfragment" version="3.0" />
-          <facet id="java" version="[1.4-1.7]" />
+         <runtime-component
+               id="com.google.appengine.standard.runtime"
+               version="1"></runtime-component>
+         <facet
+               id="com.google.cloud.tools.eclipse.appengine.facet"
+               version="1">
+         </facet>
+         <facet
+               id="jst.web"
+               version="[2.2-2.5]">
+         </facet>
+         <facet
+               id="java"
+               version="1.7">
+         </facet>
       </supported>
-      
       <adapter>
-         <runtime-component id="com.google.cloud.tools.eclipse.runtime.component.custom" />
+         <runtime-component
+               id="com.google.appengine.standard.runtime">
+         </runtime-component>
          <!-- TODO non-internal class to use here for factory -->
-         <factory class="org.eclipse.jst.server.core.internal.RuntimeClasspathProvider$Factory" />
-         <type class="org.eclipse.jst.common.project.facet.core.IClasspathProvider" />
+         <factory
+               class="org.eclipse.jst.server.core.internal.RuntimeClasspathProvider$Factory">
+         </factory>
+         <type
+               class="org.eclipse.jst.common.project.facet.core.IClasspathProvider">
+         </type>
       </adapter>
    </extension>
-   
-   <extension point="org.eclipse.jst.server.core.runtimeFacetMappings">
-    <runtimeFacetMapping runtimeTypeId="com.google.cloud.tools.eclipse.runtime.custom"
-        runtime-component="com.google.cloud.tools.eclipse.runtime.component.custom" version="1"/>
+   <extension
+         point="org.eclipse.jst.server.core.runtimeFacetMappings">
+      <runtimeFacetMapping
+            id="com.google.appengine.runtimeFacetMapping"
+            runtime-component="com.google.appengine.standard.runtime"
+            runtimeTypeId="com.google.appengine.standard.runtime"
+            version="1">
+      </runtimeFacetMapping>
    </extension>
-
-   <extension point="org.eclipse.wst.server.ui.serverImages">
+   <extension
+         point="org.eclipse.wst.server.ui.serverImages">
       <image
             icon="platform:/plugin/com.google.cloud.tools.eclipse.appengine.ui/icons/gcp-16x16.png"
-            id="com.google.cloud.tools.eclipse.image"
-            typeIds="com.google.cloud.tools.eclipse.server.custom">
+            id="com.google.appengine.serverImage"
+            typeIds="com.google.appengine.standard.runtime, com.google.appengine.server">
```

Comment:
com.google.appengine is owned by another team: https://cloud.google.com/appengine/docs/java/javadoc/
I'd prefer to avoid possible conflicts